ecs_fargate/README.md

```diff
@@ -8,65 +8,92 @@ Get metrics from all your containers running in ECS Fargate:
 
-After saving that task definition locally as `redis-datadog.json`, register the task with ECS using the [AWS CLI][11]:
+##### AWS CLI
 
+1. Download [datadog-agent-ecs-fargate.json][18].
+2. Update the json with a **TASK_NAME** and your [Datadog API Key][13].
+3. Execute the following command to register the ECS task definition:
 ```
-aws ecs register-task-definition --cli-input-json file://./redis-datadog.json
+aws ecs register-task-definition --cli-input-json file://<PATH_TO_FILE>/datadog-agent-ecs-fargate.json
 ```
 
-Once the task has been registered, you can run it in a Fargate cluster, making sure to provide one or more private subnets and security groups for your task. Note that Fargate version 1.1.0 or greater is required, so specify the platform version in the command:
+##### Web UI
+
+1. Log in to your [AWS Web Console][12] and navigate to the ECS section.
+2. Click on **Task Definitions** in the left menu, then click the **Create new Task Definition** button.
+3. Select **Fargate** as the launch type, then click the **Next step** button.
+4. Enter a **Task Definition Name**, such as `datadog-agent-task`.
+5. Select a Task execution IAM role. See permission requirements in the [Create or Modify your IAM Policy](##create-or-modify-your-iam-policy) section below.
+6. Choose **Task memory** and **Task CPU** based on your needs.
+7. Click the **Add container** button.
+8. For **Container name** enter `datadog-agent`.
+9. For **Image** enter `datadog/agent:latest`.
+10. For **Memory Limits** enter `256` soft limit.
+11. Scroll down to the **Advanced container configuration** section and enter `10` in **CPU units**.
+12. For **Env Variables**, add the **Key** `DD_API_KEY` and enter your [Datadog API Key][13] as the value. *If you feel more comfortable storing secrets in s3, refer to the [ECS Configuration guide][14].*
+13. Add another environment variable using the **Key** `ECS_FARGATE` and the value `true`.
+14. Click **Add** to add the container. Click **Create** to create the task definition.
+
+#### Create or Modify your IAM Policy
+Add the following permissions to your [Datadog IAM policy][15] to collect ECS Fargate metrics. For more information on ECS policies, [review the documentation on the AWS website][16].
+
+| AWS Permission                   | Description                                                       |
+|----------------------------------|-------------------------------------------------------------------|
+| `ecs:ListClusters`               | List available clusters.                                          |
+| `ecs:ListContainerInstances`     | List instances of a cluster.                                      |
+| `ecs:DescribeContainerInstances` | Describe instances to add metrics on resources and tasks running. |
+
+#### Run the Datadog Agent as a Replica Service
+The only option in ECS Fargate is to run the Datadog Agent as a [Replica Service][17].
```

Comment:
I'm not sure this is relevant?  We should instead indicate that the Datadog agent runs as a side care in each fargate task.

Reply:
👍 replicated services are for classic ecs, not fargate.

Reply:
When you navigate the AWS Web UI, this is option available to start the service. If someone were using ECS with EC2, Daemon service would be an option.

I will add the note about the side car to this section.